TextField/TextField.jsx

```diff
@@ -75,6 +76,19 @@ class TextFieldInput extends MaterialComponent {
   materialDom(allprops) {
     let { className, ...props } = allprops;
 
```

Comment:
@mellogarrett can we just do `className = className || ''`.
This will clean up nested conditions

Reply:
I don't know how I would write that in this case... `className = className + "..." || "..."` doesn't work

Reply:
just `className = className || ''`, below line 76 doesn't work?
className should be a simple string isnt it?

Reply:
It is a string, but if it exists we want to add to it. If it's null or undefined we want to set it. Right?

Reply:
so the code above will just initialize an empty string if its null or will leave it as is if its there

Reply:
Oh, duh... sorry, I get it now.

Reply:
I was thinking do it all on one line